.github/workflows/integration-test.yaml

```diff
@@ -5,22 +5,27 @@ on:
   - pull_request
 
 jobs:
-  test:
+  kafka-test:
     runs-on: ubuntu-latest
     steps:
-      - name: Setup Docker
-        uses: docker-practice/actions-setup-docker@master
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
       - name: Checkout repo
         uses: actions/checkout@v2
-      - name: Setup Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: "1.18"
+      - name: setup integration environment
+        uses: ./.github/workflows/integration-test-setup
```

Comment:
@turtleDev Let's inline this. Any reason to split this workflow?

Reply:
This action contains the common steps necessary for running integration tests for kafka and pubsub.
This was introduced to reduce redundant workflow steps within our testing pipelines.